cogstat/cogstat_stat.py

```diff
@@ -536,30 +537,34 @@ def variable_pair_regression_coefficients(slope, intercept, std_err, intercept_s
         Slope of the regression line
     intercept : float
         Y-intercept of the regression line
-    std_err: float
+    std_err : float
         Standard error of the slope
-    intercept_stderr: float
+    intercept_stderr : float
         Standard error of the intercept
-    meas_lev: str
+    meas_lev : str
         Measurement level of variables
-    n: int
+    n : int
         Number of data points
-    normality: bool
+    normality : bool
         True if variables follow a multivariate normal distribution, False otherwise. None if normality couldn't be
         calculated or if the parameter was not specified.
+    homoscedasticity : bool
+        True if variables are homoscedastic, False otherwise. None if homoscedasticity couldn't be calculated or
+        if the parameter was not specified.
 
     Returns
     -------
     str
         Table of the point and interval estimations
     """
+    print(homoscedasticity)
     if meas_lev == "int":
         regression_coefficients = '<cs_h3>' + _('Regression coefficients') + '</cs_h3>'
         pdf_result = pd.DataFrame(columns=[_('Point estimation'), _('95% confidence interval')])
 
+        # Warinings based on the results of the assumption tests
         if normality is None:
-            regression_coefficients += '\n' + '<decision>' + _('Normality could not be calculated.') + ' ' + \
-                                       _('CIs may be biased.') + '</decision>'
+            regression_coefficients += '\n' + '<decision>' + _('Normality could not be calculated!') + '</decision>'
```

Comment:
I'm not sure if the exclamation mark is easy to interpret for the user. And if we cannot calculate the normality, then there's a risk that the CIs are biased. So I'd move back to the original message.

Reply:
I reverted back to original message. The logic was that in certain cases, if normality cannot be calculated, then CIs don't get calculated either, so the message isn't relevant to the user.